index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <script src="app.js"></script>
+    <script src="app.js" defer></script>
```

Comment:
why defer can you explain @Abbas4tech ?

Reply:
When the browse hits the `script` tag , defer downloads the script file simultaneously parsing the html and only executes it when the parsing of html completed.